scripts/retrieveDownloadLink.js

```diff
@@ -1,4 +1,4 @@
-const TARGET_VERSION = '81.0.416.72';
+const TARGET_VERSION = '83.0.478.37';

```

Comment:
83.0.478.45 is the one we have on thirdpartysource

Reply:
thanks @mliao95 .